File: Java/UberBlack.java
import java.util.ArrayList;
import java.util.Map;

class UberBlack extends Car {
  Map<String, Map<String, Integer>> typeCarAccepted;
  ArrayList<String> seatsMaterials;

  public UberBlack(String license, Account driver, Map<String, Map<String, Integer>> typeCarAccepted,
      ArrayList<String> seatsMaterials) {
    super(license, driver);
    this.typeCarAccepted = typeCarAccepted;
    this.seatsMaterials = seatsMaterials;
  }
}

File: Python/UberBlack.py
from car import Car

class UberBlack(Car):
  typeCarAccepted = []
  seatsMaterials = []

  def __init__(self, license, driver, typeCarAccepted, seatsMaterials):
    super.__init__(license, driver)
    self.typeCarAccepted = typeCarAccepted
    self.seatsMaterials = seatsMaterials